domains/pallets/domain-registry/src/lib.rs

```diff
@@ -569,14 +572,33 @@ mod pallet {
                     log::error!(target: "runtime::domain-registry", "Bad core bundle, error: {e:?}");
                     TransactionValidityError::Invalid(InvalidTransactionCode::Bundle.into())
                 }),
-                Call::submit_fraud_proof { .. } => Ok(()),
+                Call::submit_fraud_proof { fraud_proof } => {
+                    if !fraud_proof.domain_id().is_core() {
```

Comment:
It's okay for now, but it can be more future-proof if we check whether it targets to system domain since the open domain is also possible later.

```rust
if fraud_proof.domain_id().is_system() {
    Err(...)
} else {
    Ok(())
}
```